agent/taskresource/volume/volume.go

```diff
@@ -98,11 +105,42 @@ func (vol *VolumeResource) GetCreatedAt() time.Time {
 	return vol.createdAtUnsafe
 }
 
+// Create performs resource creation
+func (vol *VolumeResource) Create() error {
+	volumeResponse := vol.client.CreateVolume(
+		vol.Name,
+		vol.Driver,
+		vol.DriverOpts,
+		vol.Labels,
+		engine.CreateVolumeTimeout)
+
+	if volumeResponse.Error != nil {
+		seelog.Criticalf("Unable to create volume %s with driver %s: %v", vol.Name, vol.Driver, volumeResponse.Error)
```

Comment:
wdyt about removing this log line here? I think logging it at the task engine/managed task level makes more sense as we'll also be able to emit the task id up there. Either that, or we should inject the task id/arn here.